packages/wonder-blocks-testing/src/harness/adapters/data.js

```diff
@@ -0,0 +1,51 @@
+// @flow
```

Comment:
Copied verbatim from webapp, but renamed to just `data` instead of `wonderBlocksData`.